vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh

```diff
@@ -1,5 +1,15 @@
 #!/bin/bash
 
+{{/* FIPS-related error codes */}}
```

Comment:
can we move the error codes to cse_helper.sh?

Reply:
moved the error codes to cse_helper.sh.